src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -582,6 +582,15 @@ public BoundExpression StaticCall(TypeSymbol receiver, MethodSymbol method, para
             return Call(null, method, args);
         }
 
+        public BoundExpression StaticCall(TypeSymbol receiver, WellKnownMember method, params BoundExpression[] args)
+        {
+            MethodSymbol methodSymbol = WellKnownMethod(method);
+            Debug.Assert(methodSymbol.IsStatic);
+            Debug.Assert(receiver.GetMembers(methodSymbol.Name).OfType<MethodSymbol>().Contains(methodSymbol));
+            Binder.ReportUseSiteDiagnostics(methodSymbol, Diagnostics, Syntax);
```

Comment:
The call to ReportUseSiteDiagnostics turns out to be necessary. WellKnownMethod does not report the diagnostics. Maybe it should.